.github/actions/spell-check/expect.txt

```diff
@@ -752,25 +668,21 @@ hotkeys
 hotlight
 hotspot
 HPAINTBUFFER
-HPALETTE
 HRAWINPUT
-hread
 HREDRAW
 hres
 hresult
 hrgn
-HRSRC
 hsb
 HSCROLL
 hsi
-hsl
 hstring
-hsv
 htcfreek
```

Comment:
Shouldn't this and similar entries be in names.txt?

Reply:
That's exactly what I wanted to do 😉